src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,52 @@
+package duke.parser;
+
+import duke.command.AddDeadline;
+import duke.command.AddEvent;
+import duke.command.AddTodo;
+import duke.command.Bye;
+import duke.command.Command;
+import duke.command.Delete;
+import duke.command.Find;
+import duke.command.List;
+import duke.command.Remark;
+import duke.command.Unknown;
+
+/**
+ * A utility class for parsing user input and creating corresponding command objects.
+ * This class processes user input to determine the type of command to create.
+ */
+public class Parser {
+    /**
+     * Parses the user input and creates a corresponding command object.
+     *
+     * @param instr The user input string.
+     * @return A command object based on the parsed input.
+     */
+    public static Command parse(String instr) {
+        String keyWord = instr.split(" ")[0];
+        switch (keyWord) {
+        case"":
```

Comment:
i like the use of switch-case and the correct coding standard applied according to this course!